docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -708,11 +708,19 @@ Backend     SQL Equivalent
 ==========  ======================================
 PostGIS     ``ST_DWithin(poly, geom, 5)``
 Oracle      ``SDO_WITHIN_DISTANCE(poly, geom, 5)``
+SpatiaLite  ``PtDistWithin(poly, geom, 5)``
 ==========  ======================================
 
 .. note::
+    The default units being queried against will vary by backend. On PostGIS
```

Comment:
"default" suggests to me that there's some way to customize it -- is it the case?

This can be a regular paragraph rather than a note box, above the "Example::" above.